OSDOCS-4660: Add no-update warning to About page (#53837)
openshift/openshift-docs: welcome/index.adoc

Start with **xref:../architecture/architecture.adoc#architecture-overview-architecture[Architecture]**.
endif::[]

ifdef::microshift[]

> Start with xref:../microshift_getting_started/microshift-understanding.adoc#microshift-understanding[Understanding {product-title}] and

Reviewer: We aren't supposed to hard wrap lines. Would you mind undoing the hard wraps here before we merge?

xref:../microshift_install/microshift-install-rhel-for-edge.adoc#microshift-install-rhel-for-edge[Installing].
Then, see the
xref:../microshift_release_notes/microshift-4-12-release-notes.adoc#microshift-4-12-release-notes[release notes].

[IMPORTANT]
====

> Red Hat will not provide or support an update or upgrade path from Developer Preview or Technology Preview versions to later versions of {product-title}. A new installation will be necessary.

Reviewer: This sentence is really complex with the amount of "ors" there are in the sentence (3).

Could we simplify this a bit and keep the same sentiment? Also, is it necessary to even mention TP right now since it's not even been provided as TP yet?

```suggestion
Red Hat will not support updating from Developer Preview versions to later versions of {product-title}. A new installation will be necessary.
```

Author: I think this edit isn't as explicit as we need, but I will change that third "or" to an "and", split the difference a bit.

> Red Hat might not provide or support an update or upgrade path from Developer Preview or Technology Preview versions to the Generally Available version of {product-title}. A new installation might be necessary.

Reviewer 2: ```suggestion
Red Hat will not provide or support an update or upgrade path from Developer Preview or Technology Preview versions to later versions of {product-title}. A new installation will be necessary.
```

@DanielFroehlich I'm proposing some stronger language here, changing "might not" to "will not". Can you sign off on that?

@ShaunaDiaz I changed "the Generally Available version" to "later versions" because we wouldn't support upgrading from dev preview to tech preview either. I tried to find wording to be more explicit, but that made the sentence unparseable. If you have other wording that conveys the same meaning, that's fine with me.

Reviewer 3: /lgtm
